src/any_support.cc

```diff
@@ -130,10 +149,14 @@ LIBRARY_EXPORT int32_t TryUnpackFromAny(grpc_labview::gRPCid* id, grpc_labview::
 //---------------------------------------------------------------------
 //---------------------------------------------------------------------
 LIBRARY_EXPORT int32_t IsAnyOfType(grpc_labview::gRPCid* id, grpc_labview::AnyCluster* anyCluster, const char* messageType)
-{    
+{
     auto metadataOwner = id->CastTo<grpc_labview::IMessageElementMetadataOwner>();
-    auto metadata = metadataOwner->FindMetadata(messageType);
+    if (!metadataOwner)
+    {
+        return -1;
+    }
 
+    auto metadata = metadataOwner->FindMetadata(messageType);
     if (grpc_labview::GetLVString(anyCluster->TypeUrl) != metadata->typeUrl)
```

Comment:
This function should check `metadata`.

I noticed this because of the "return -1, return -2" vs. "return -1, return -1"

Reply:
I added a null check here as well. The error codes here don't really matter as the return type at the VI layer API just converts this to a Boolean for success or failure.